### PR TITLE
Urlscan BugFixes

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
Ready

## Related Issues
fixes: demisto/etc#17768
fixes: demisto/etc#19044

## Description
- Enabled global override of the public/private argument via a parameter.
- URLScan now uses the `verdicts` entry in the response.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Integrations/UrlScan/URLScan.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def poll(target, step, args=(), kwargs=None, timeout=None, max_tries=None, check
 
 def urlscan_submit_url():
     submission_dict = {}
-    if demisto.args().get('public') == 'public':
+    if demisto.params().get('privacy') == 'public':
```
Comment (yuvalbenshalom) [inline Integrations/UrlScan/URLScan.py]: Seems like you disabled the argument of the command.
We should talk about it again because this breaks BC anyway.

**Comment by amshamah419** [comment]
> we should think again on the public/private defaults because it will break BC anyway.
> and seems like WHOIS files are in this PR by mistake.

I think I know what happened here... In my mind, I already merged the Whois stuff. I did not yet. Checking with Alex to see if we can get a green light on docs. Then the diff should be the same.

File: Integrations/UrlScan/URLScan.py (hunk 1 above)
Comment (amshamah419) [reply Integrations/UrlScan/URLScan.py]: I agree. I'm not sure how we would both change the default and not break BC. I could change the argument do do something like:
```
if demisto.args().get('public') == 'public' and demisto.params().get('override) is not True:
   do the thing
```

File: Integrations/UrlScan/URLScan.yml [hunk 2]
```diff
@@ -28,9 +28,15 @@ configuration:
   defaultvalue: "1"
   type: 0
   required: false
+- display: Enable public submissions by default.
+  name: is_public
+  defaultvalue: false
```
Comment (yuvalbenshalom) [inline Integrations/UrlScan/URLScan.yml]: change to true

File: Integrations/UrlScan/URLScan_CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+  - URLScan now leverages the _Verdict_ result from the API.
```
Comment (yuvalbenshalom) [inline Integrations/UrlScan/URLScan_CHANGELOG.md]: add comment about the new parameter

**Comment by amshamah419** [comment]
@kirbles19 Can you review these? 
https://github.com/demisto/content/blob/urlscan-bf/Integrations/UrlScan/URLScan_CHANGELOG.md

https://github.com/demisto/content/blob/urlscan-bf/Integrations/UrlScan/URLScan.yml

**Comment by kirbles19** [comment]
@amshamah419 looks like this integration needs updates for context standards.

**Comment by amshamah419** [comment]
@kirbles19 Looks like they were there in the code, just not in the yml. I updated the yml to have the context outputs.

**Comment by kirbles19** [comment]
@amshamah419 All done.

**Pull request merged**